npm/rules.bzl

```diff
@@ -74,7 +74,7 @@ assemble_npm = rule(
             cfg = "host"
         ),
         "_npm": attr.label(
-            default = Label("@nodejs//:bin/npm"),
+            default = Label("@nodejs//:node"),
```

Comment:
This is a migration change necessary for `rules_nodejs`. Likely, the `assemble_npm` and `deploy_npm` rules need to change entirely, but we are not in a position to test this yet.